### PR TITLE
docs: add TiKV Control User Guide

### PR DESCRIPTION
**Pull request by lilin90**
Signed-off-by: lilin90 <lilin@pingcap.com>

## What have you changed? (mandatory)

Add a document: TiKV Control User Guide

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

From docs: https://github.com/pingcap/docs/blob/master/tools/tikv-control.md

## Does this PR affect tidb-ansible update? (mandatory)

No.

### CODE REVIEW

**Comment by lilin90** [comment]
@Connor1996 @hicqu @CaitinChen PTAL

File: docs/tikv-control.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+> **Note:**
+> 
+> - This command only supports the local mode. It prints `success!` when successfully run.
+> - You must run this command for all stores where specified Regions' peers locate. If `--regions` is not set, all Regions are involved, and you need to run this command for all stores.
+
+### Recover from MVCC data corruption
+
+Use the `recover-mvcc` command in circumstances where TiKV cannot run normally caused by MVCC data corruption. It cross-checks 3 CFs ("default", "write", "lock") to recover from various kinds of inconsistency.
+
+Use the `--regions` option to specify involved Regions by `region_id`. Use the `--pd`option to specify PD endpoints.
```
Comment (Connor1996) [inline docs/tikv-control.md]: please add a space after `--pd`

**Comment by hicqu** [comment]
LGTM.

**Comment by siddontang** [comment]
Please approve it @hicqu @Connor1996

File: docs/tikv-control.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+```bash
+$ tikv-ctl --db /path/to/tikv/db unsafe-recover remove-fail-stores --stores 3 --regions 1001,1002
+success!
+```
+
+> **Note:**
+> 
+> - This command only supports the local mode. It prints `success!` when successfully run.
+> - You must run this command for all stores where specified Regions' peers locate. If `--regions` is not set, all Regions are involved, and you need to run this command for all stores.
```
Comment (Connor1996) [inline docs/tikv-control.md]: locate -> are located

**Pull request merged**